src/main/java/com/dynatrace/prototype/payloadHandler/SlackPayloadHandler.java

```diff
@@ -0,0 +1,43 @@
+package com.dynatrace.prototype.payloadHandler;
+
+import com.slack.api.Slack;
+import com.slack.api.methods.SlackApiException;
+import com.slack.api.methods.request.chat.ChatPostMessageRequest;
+import com.slack.api.methods.response.chat.ChatPostMessageResponse;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.io.IOException;
+
+@ApplicationScoped
+public class SlackPayloadHandler implements PayloadHandler {
+    private static final String ENV_SLACK_TOKEN = "SLACK_TOKEN";
+    private static final String ENV_SLACK_CHANNEL = "SLACK_CHANNEL";
+
+    @Override
+    public boolean sendPayload(String payload) {
+        boolean messageSent = false;
+        Slack slack = Slack.getInstance();
+        String token = System.getenv(ENV_SLACK_TOKEN);
+        String channel = System.getenv(ENV_SLACK_CHANNEL);
```

Comment:
I would print an error on `System.err` if these env vars are missing

Reply:
I don't think this is necessary because Slack  throws an exception if the token / channel is wrong / missing (and its stack trace will be printed bellow.